util_test.go

```diff
@@ -374,3 +374,40 @@ func AlwaysRNG() *TestRNG {
 func SimpleRNG() *TestRNG {
 	return &TestRNG{rolls: []bool{true, false}}
 }
+
+// Create a single Battle. A single battle is a battle where only 1 pokemon is sent out at a time, and there can only be 2 parties.
+// This function should only be used in tests.
+func NewSingleBattle(p1 *Party, a1 *Agent, p2 *Party, a2 *Agent) *Battle {
+	b := NewBattle()
+	b.AddParty(p1, a1, 0)
+	b.AddParty(p2, a2, 1)
+	return b
+}
+
+// Create a single Battle with only 1 pokemon in each party.
+// This function should only be used in tests.
+func New1v1Battle(p1 *Pokemon, a1 *Agent, p2 *Pokemon, a2 *Agent) *Battle {
+	return NewSingleBattle(NewOccupiedParty(p1), a1, NewOccupiedParty(p2), a2)
+}
+
+// Creates a new party to store Pokemon and assigns them to a team
+// This function should only be used in tests.
+func newBattlePartyOld(agent *Agent, team int) *battleParty {
```

Comment:
why is this function "old"? is it deprecated?

Reply:
Yeah, I'll mark it as such